File: 백준/Bronze/16917. 양념 반 후라이드 반/양념 반 후라이드 반.cs
using System;

class Program
{
    static void Main() {
        int a,b,c,x,y,price = 0;
        string[] input;
        input = Console.ReadLine().Split(' ');
        a = int.Parse(input[0]);
        b = int.Parse(input[1]);
        c = int.Parse(input[2]);
        x = int.Parse(input[3]);
        y = int.Parse(input[4]);
        
        if(a+b > c*2)
        {
            for(int i=0;i<Math.Min(x,y);i++) price = price + (c * 2);

            if(a > c*2 && x > y) price = price + c*2*(x-y);
            else if (b > c*2 && x < y) price = price + c*2*(y-x);
            else price = price + (x > y ? a : b)*(Math.Max(x,y) - Math.Min(x,y));
        }
        else price = a*x+b*y;
        Console.Write(price);
    }
}

File: 백준/Bronze/16917. 양념 반 후라이드 반/양념 반 후라이드 반.py
a,b,c,x,y = map(int,input().split())
price = 0

if a+b > c*2 :
    for i in range(min(x,y)): price = price + (c * 2);

    if a > c*2 and x > y: price = price + c*2*(x-y);
    elif b > c*2 and x < y: price = price + c*2*(y-x);
    else: price = price + (a if x > y else b)*(max(x,y) - min(x,y));
else: price = a*x + b*y;

print (price);